specs/_features/eip7732/fork-choice.md

```diff
@@ -628,14 +559,43 @@ def on_payload_attestation_message(
     time_into_slot = (store.time - store.genesis_time) % SECONDS_PER_SLOT
     if is_from_block and time_into_slot >= SECONDS_PER_SLOT // INTERVALS_PER_SLOT:
         return
+```
```

Comment:
The comment in line 556 about updating payload boosts need to be gone.  The function can return early right after line 554